docs/provisioning/external-silos.md

```diff
@@ -0,0 +1,154 @@
+# Advanced - Provisioning a setup with _external_ silos
```

Comment:
Could you link it from the README.md in docs/provisioning/ ?

Reply:
done